content/en/upcoming-features.md

```diff
@@ -1,52 +1,52 @@
 ---
 title: Upcoming Features
 slug: upcoming-features
-lastmod: 2024-06-14
+lastmod: 2025-05-13
 show_lastmod: 1
 ---
 
 For announcements of upcoming changes, please see the [API Announcements](https://community.letsencrypt.org/c/api-announcements/18) category on the Let's Encrypt community forum.
```

Comment:
Want to tell people to subscribe to the Technical Updates mailing list? could link to /opt-in/ page

Reply:
Good call, done